INSTALL.md

```diff
@@ -30,37 +28,52 @@ make
 ./porymap.app/Contents/MacOS/porymap
 ```
 
-## Windows
-
+## Windows 7+
 Install [Qt development tools](https://www.qt.io/download-qt-installer), and use Qt Creator, the official Qt IDE, for development purposes.
 
-## Ubuntu
 
-You need to install Qt. The minimum supported version is currently Qt 5.14.2. You can check your Qt version
-with `qtdiag` or `qmake --version`.
+## Gentoo Linux
+You need to install Qt dev tools. You can set version of your Qt packages with flags such as `qt6`.
```

Comment:
It could be that I'm missing context here, but this seems unclear (flags provided to what? and how?)